src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -21,14 +23,26 @@ const httpError = require('http-errors')
 const parseUrl = require('parseurl')
 const finalHandler = require('finalhandler')
 const bodyParser = require('body-parser')
+const sendFile = require('send')
+
+const { POSTGRAPHQL_ENV } = process.env
 
 const debugGraphql = new Debugger('postgraphql:graphql')
 const debugRequest = new Debugger('postgraphql:request')
 
+const graphiqlDirectory = resolvePath(__dirname, '../graphiql/public')
+
 const favicon = new Promise((resolve, reject) => {
   readFile(resolvePath(__dirname, '../../../resources/favicon.ico'), (error, data) => {
     if (error) reject(error)
-    else resolve(data)
+    else resolve(data.toString())
+  })
```

Comment:
Why are you converting your favicon to a `utf8` string here? I think you actually want a `Buffer` here; using `fs-promise` it'd just be:

```
const favicon = fspromise.readFile(resolvePath(...))
```